components/ui-api-layer/docs/proposals/exceed-resource-quota.md

```diff
@@ -0,0 +1,189 @@
+# Check when the ResourceQuota is exceeded
+
+## Overview
+
+[ResourceQuota](https://kubernetes.io/docs/concepts/policy/resource-quotas/) provides constraints that limit aggregate resource consumption per Namespace.
+
+The ResourceQuota values can be as follows:
+
+```bash
+Name:            kyma-default
+Resource         Used       Hard
+--------         ---        ---
+limits.memory    7348440Ki  10Gi
+requests.memory  3727576Ki  7Gi
+pods             6          10
+resourcequotas   1          1
+```
+
+The Console calls for the ResourceQuota status. The calls are triggered after: 
+- switching the environment
+- uploading a resource
+- creating a lambda
+- open environment details
+
+There are a few ways to check if the ResourceQuota exceeds its limit.
+
+## Expose the ResourceQuotaStatus
+
+The idea is to create an endpoint which returns the ResourceQuotaStatus.
+ResourceQuotaStatus contains the flag. The value of this flag informs you if the ResourceQuota is exceeded. 
+
+In case when the `used` value equals the `hard` value, the flag is set to true so the ResourceQuota is exceeded.
```

Comment:
equals or is bigger than (not sure if it can be a case but it will be safer assumption)